README.md

```diff
@@ -12,6 +12,7 @@ Both Social Team members and Node.js project collaborators submitting PRs should
 - We do not amplify or share content that could unduly promote or benefit one corporation that does work in the space of Node.js over another.
 - Where possible, we try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
   - As an example, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
+  - As an exception, in the case where important or potentially negative content is being shared, using the main account's voice is preferable to both reduce potential negative impact on the indivduals we'd be amplifying and increase the message's signal to end-users (seeing someone you don't know vs. seeing the account have different levels of impact).
```

Comment:
```suggestion
- We do not amplify or share content that could unduly promote or benefit one corporation that does work in the space of Node.js over another.
- Where possible, we try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
  - For general releases, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
  - For security releases, it is important that the tweet be from an authoritative source that everyone will immediately recognize. So announcements for those releases are made directly from the project account.
```

Reply:
I'm happy with @Trott's versions instead of what I had suggested.

Reply:
I am happy to have this specific policy but I also would like to codify what I initially wrote:

- in situations where a collaborator could be targeted / harassed for simply sharing a subject (I've seen this happen) that we'd want to amplify from the social accounts, I'd prefer that we post from the social accounts directly rather than _potentially_ enabling harassment of our maintainers
- in situations where a message is important/sensitive, tweeting from the official account as an authoritative source that can be trusted is important.

The above edits address a single instance of the second point but don't address it generally, and completely omit the first point.

Reply:
> The above edits address a single instance of the second point but don't address it generally, and completely omit the first point.

Maybe we can use some of your first bullet point nearly verbatim like this?

```markdown
- We try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
  - For general releases, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
  - For security releases, it is important that the tweet be from an authoritative source that everyone will immediately recognize. So announcements for those releases are made directly from the project account.
- In situations where a collaborator might become a target for harassment, post from the project account directly. Examples include _lorem ipsum_.
```

The last "Examples include" sentence is optional. Use whatever examples are appropriate in the last sentence or leave it off entirely. (It can always be added later.)